.npmignore

```diff
@@ -1,11 +1 @@
-.babelrc
-.flowconfig
-.eslintrc
-.gitignore
-.github/*
-.circleci/*
-jsconfig.json
-commitlint.config.js
-changelog.js
-assets/*
```

Comment:
I think we should keep this and not publish them to npm

Reply:
Correct, it should not be published. Please also check the updated `files` section in `package.json` - It works in tandem with npmignore, and declares files that are _included_.